### ISSUE TITLE
Error: module 'PySide6QtAds' has no attribute 'ads'

### ISSUE THREAD

**Issue by foongminwong** [mborgerson/pyside6_qtads#16]
_Moving issue from https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/issues/508_

Hello, I got this error when running [simple main.py](https://github.com/mborgerson/Qt-Advanced-Docking-System/blob/pyside6/examples/simple/main.py) and other examples.

```
PS E:\Qt-Advanced-Docking-System\examples\simple> python .\main.py
Traceback (most recent call last):
  File ".\main.py", line 52, in <module>
    w = MainWindow()
  File ".\main.py", line 26, in __init__
    self.dock_manager = QtAds.CDockManager(self)
AttributeError: Error evaluating `PySide6QtAds.ads.CDockManager.__init__`: module 'PySide6QtAds' has no attribute 'ads'
```

I came across the previous issues https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/issues/298 and https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/issues/494, upgraded `PySide6-QtAds` to 4.0.1.2, and it doesn't seem to fix the error on my end...

**Environment:**
OS: Windows Server 2019 Standard
Python Version: 3.8.16
Package Version: 
![image](https://user-images.githubusercontent.com/23441795/233530008-1fdef659-4ef1-4031-a8df-e714ef0caecc.png)
![image](https://user-images.githubusercontent.com/23441795/233531069-b33449dc-cf85-4ef6-bc1e-691d6289dc2a.png)

**Relevant Information**
I also received this error when I installed the latest PySide6, am not sure if this might cause the error... Does it mean I have to downgrade ` pyside6-essentials` & `shiboken6`?
![image](https://user-images.githubusercontent.com/23441795/233532660-4608d538-a7a3-4add-b8c5-26317bde06e8.png)

**Comment by demotu** [comment]
I should probably repeat my answer here in case people get here with this issue...

As shown in your pip error message, at the time you tried, PySide6-QtAds 4.0.1.2 required PySIde6 6.4.3, but you had PySide6 6.5.0 installed (and the equivalent versions for essentials and shiboken).

But PySide6-QtAds 4.0.3 was just released and it is compatible with PySide6 6.5.0. See https://pypi.org/project/PySide6-QtAds/

**Comment by foongminwong** [comment]
Just reinstalled `PySide6-QtAds 4.0.3` and it works, thank you!
